website/docs/quick-start-guide.md

```diff
@@ -778,18 +724,20 @@ val updates = convertToStringList(dataGen.generateUpdates(10))
 val df = spark.read.json(spark.sparkContext.parallelize(updates, 2))
```

Comment:
Understand there is a note below the code snippet, on spark datasource not supporting updates/deletes on keyless table. It might be good to call out in a comment line at the start of the snippet, that, this update is on primary keyed table only as a description line.